.travis/dispatch.sh

```diff
@@ -72,4 +72,10 @@ else
     if [[ "${SALT_NODE_ID}" == "servo-master1" ]]; then
         ./test.py sls.buildbot.master sls.homu sls.nginx
     fi
+
+    # Salt doesn't support timezone.system on OSX
+    # See https://github.com/saltstack/salt/issues/31345
+    if [[ "${TRAVIS_OS_NAME}" != "osx" ]]; then
```

Comment:
Sorry about that. I thought it would be less error-prone (because the servo-mac* could change at some point) but if it needs to be travis independant I totally get that.